stdump.cpp

```diff
@@ -130,16 +130,27 @@ void print_types(Program& program, SymbolTable& symbol_table) {
 					prefix += sym.string.substr(0, sym.string.size() - 1);
 				} else {
 					const std::string full_symbol = prefix + sym.string;
-					printf("*** PARSING %s\n", full_symbol.c_str());
 
-					const StabsSymbol stab_sym = parse_stabs_symbol(full_symbol.c_str());
+					if(verbose)
+						printf("*** PARSING %s\n", full_symbol.c_str());
+
+					const StabsSymbol stab_sym = parse_stabs_symbol(full_symbol.c_str(), verbose);
 					const std::string sym_name = stab_sym.name;
 
 					if (declared_symbols.count(sym_name) == 0) {
+						auto aggregate_max_name_size = [](const auto& fields) -> size_t {
```

Comment:
accidentally put this in the wrong commit while rebasing, sorry. hopefully not too much of an issue